.github/SECURITY.md

```diff
@@ -0,0 +1,44 @@
+# Security policy
+
+1. [Supported Versions](#1-supported-versions)  
+2. [Vulnerability Report](#2-vulnerability-report)  
+3. [Security Disclosure](#3-security-disclosure)  
+
+---
+
+## 1. Supported Versions
+
+We are releasing patches to eliminate vulnerabilities, you can see below:
+
+| Version     | Supported by |
+| ----------- | ------------ |
+| 1.0.x       | N/A          |
+
+---
+
+## 2. Vulnerability Report
+
+The Edge-Orchestration team assigns the highest priority to all security bugs in Edge-Orchestration project. We appreciate your efforts and
+responsible disclosure of information to eliminate vulnerabilities.
+
+Please report security bugs by emailing the Lead Maintainer at moonki1.hong@samsung.com or Security Advisor at t.drozdovsky@samsung.com marked "SECURITY" or create an issue in the repository with start title "SECURITY".
+Our team will confirm your request and within 72 hours will try to prepare recommendations for elimination. Our team will keep you updated on the progress towards the fix until the full announcement of the patch release. During this process, the edge-orchestration team may request additional information or guidance.
```

Comment:
What about soften the deadline larger than 72 hours? We understand that 3-day review period is good enough in the open source community, but it sometimes makes a lot of undesired pressure on the task regarding the review processes.

Reply:
I don't mind, suggest your version please

Reply:
What about 1 week?

Reply:
Best practices offer 48 hours of response time when they accept a request.
But I agree for a week. Please note that response time is a our reputation.